templates/base_index.html

```diff
@@ -474,11 +427,8 @@ <h2 class="p-heading--1 u-no-margin--bottom">Public cloud optimisation</h2>
         <div class="col-3 col-start-large-7 col-medium-2 col-medium-start-3">
```

Comment:
There is some top padding on the last row of list items. Can you please add "u-no-padding--top" to the list items on lines 417 and 420?